check-before-deployment.md

```diff
@@ -221,6 +237,48 @@ To check whether the NTP service is installed and whether it synchronizes with t
         Unable to talk to NTP daemon. Is it running?
         ```
 
+3. Run the `chronyc tracking` command to check wheter the Chrony service synchronizes with the NTP server.
+
+    > **Note:**
+    >
+    > This only applies to system that use Chrony instead of NTPd.
```

Comment:
```suggestion
    > This only applies to system that uses Chrony instead of NTPd.
```

Reply:
I think the original is sentence is correct.

Reply:
Grammatically, the logical subject of the verb "use" is "system", which is a singular noun. Therefore, in this case, "uses" is grammatically correct. If we want to use "use", "system" should be in the plural form "systems".

Reply:
Hmz. in that case both look wrong to me.

Would any of these be ok?

```suggestion
    > This only applies to systems that use Chrony instead of NTPd.
```

```suggestion
    > This only applies to a system that uses Chrony instead of NTPd.
```

Reply:
Yes. Both LGTM